flax/linen/fp8_ops.py

```diff
@@ -123,6 +123,24 @@ def out_qdq_bwd(compute_dtype, res, g):
 out_qdq.defvjp(out_qdq_fwd, out_qdq_bwd)
 
 
+@partial(custom_jvp, nondiff_argnums=(2,))
+def dot_general_with_precision(lhs, rhs, dimension_numbers):
```

Comment:
Can you make this have the same signature of the lax.dot_general() so that I can inject it to the jnp.einsum?
```
def dot_general_with_precision(lhs, rhs, dimension_numbers, precision=None, preferred_element_type=None):
   return lax.dot_general(lhs, rhs, dimension_numbers, precision=lax.Precision.DEFAULT, preferred_element_type=preferred_element_type)
  
```